index.js

```diff
@@ -76,20 +77,39 @@ exports.init = function (ssb, config) {
     }
 
     pull(
+      // we don't care about groups we've been excluded from
       ssb.tribes2.list({ live: true }),
+      pull.unique('id'),
```

Comment:
hmm shouldn't be unique necessarily, since we want to hear about new readKeys

Reply:
removing this even if it's slightly inefficient, simplest solution

Reply:
mm although could still be nice to only have 1 object per group, then e.g. excluded people would stop replicating immediately

Reply:
still, i guess mostly an optimization